src/widget/form/genericchatform.cpp

```diff
@@ -728,7 +728,7 @@ bool GenericChatForm::loadHistoryFrom(const QDateTime &time)
 
 void GenericChatForm::removeFirstsMessages(const int num)
 {
-    if (messages.size() > num) {
+    if (static_cast<int>(messages.size()) > num) {
```

Comment:
Note that signed overflow has undefined behaviour. Can you mention in the PR description why you chose this direction?